eng/pipelines/templates/jobs/archetype-sdk-tests-jobs.yml

```diff
@@ -72,7 +72,8 @@ jobs:
 
       - template: /eng/common/TestResources/deploy-test-resources.yml
         parameters:
-          Location: ${{ coalesce(parameters.Location, parameters.CloudConfig.Location, 'westus2') }}
+          ${{ if or(parameters.Location, parameters.CloudConfig.Location) }}:
```

Comment:
Are you sure DevOps treats these variables as conditionals correctly?

Reply:
I tested it with parameters.Location set, parameters.CloudConfig.Location set, and none set, and it worked in all three. Happy to change it if it's too "clever." My thinking was a longer expression would be harder to read.

Reply:
I'm fine with it as long as it works.